config/default.yml

```diff
@@ -553,6 +553,10 @@ Style/TrivialAccessors:
     - to_s
     - to_sym
 
+Style/UnneededPercentX:
+  # Enforce using %x when the command string spans multiple lines.
+  EnforcePercentXForMultiLine: false
```

Comment:
Sorry for abandoning this PR for the better part of 3 weeks :-/

I think the 3-style approach (`backticks`, `percent_x`, `mixed`) is the way to go. Then we just need a a configuration option for what to do when a backtick’ed command literal contains a backtick: Allow it, or change to use `%x`?

I was thinking (since @bbatsov mentioned #1020) that maybe the configuration option should be `MaxBackticks`, and work in the same way as `MaxSlashes` does for `Style/RegexpLiteral`?

Or maybe `Style/RegexpLiteral` should have `MaxSlashes` removed and instead have an option to allow or disallow slashes in a shash’ed regex?

Either way, I think it makes sense to have these two cops to work, and be configured, similarly.